ballerina/Module.md

```diff
@@ -128,13 +128,13 @@ The most efficient way to receive messages is to set up a subscription using a B
         queueName: "MyQueue"
     }
     service rabbitmq:Service on channelListener {
-        remote function onRequest(rabbitmq:Message message) returns string {
+        remote function onRequest(rabbitmq:BytesMessage message) returns string {
```

Comment:
Shouldn't this be `AnydataMessage`? Check the README.md and here, they seems different.